website_sale_product_minimal_price/controllers/main.py

```diff
@@ -52,7 +18,15 @@ def get_combination_info_pricelist_atributes(self, product_id, **kwargs):
         """Special route to use website logic in get_combination_info override.
         This route is called in JS by appending _website to the base route.
         """
-        pricelist = request.env["website"].get_current_website().get_current_pricelist()
+        # Copied from _get_combination_info
+        # /odoo/addons/website_sale/models/product_template.py
+        website = (
+            # pylint: disable=W8121
+            request.env["website"]
+            .get_current_website()
+            .with_context(request.env.context)
```

Comment:
Just follow the linter recommendation

Reply:
Please check comment above. I just search a current website in the same way as it made in Odoo CE
https://github.com/odoo/odoo/blob/dd97f1aa0b5a0f4e87be663cc7f2f610ee052e4b/addons/website_sale/models/product_template.py#L392

Reply:
You simply need to do this:
```suggestion
            request.env["website"]
            .get_current_website()
            .with_context(**request.env.context)
```

Reply:
Fixed